distribution/scenes/textures/patterns/crackle_solid.pov

```diff
@@ -2,97 +2,190 @@
 // To view a copy of this license, visit http://creativecommons.org/licenses/by/3.0/
 // or send a letter to Creative Commons, 444 Castro Street, Suite 900, Mountain View,
 // California, 94041, USA.
-
-// Persistence Of Vision raytracer sample file.
-// This file demonstrates some uses of the "crackle solid" pattern,
-// with a fake-granite tiled floor, and a zinc bucket.
 //
-// -w320 -h240
+// Persistence Of Vision Ray Tracer ('POV-Ray') sample file.
+//
+// Crackle pattern example crackle_solid.pov.
+//
 // -w800 -h600 +a0.3
 
-#version 3.7;
-
+#version 3.71;
 global_settings { assumed_gamma 1.0 }
+#default { finish { ambient 0.006 diffuse 0.456 } }
 
-#include "colors.inc"
-#include "woods.inc"
-
-camera { location <70,100,60> 
-         right   x*image_width/image_height
-         angle 40 
-         look_at <0,15,0>
-       }
+camera {
+    location <70,100,60>
+    right    x*image_width/image_height
+    angle    40
+    look_at  <0,15,0>
+}
 
+#declare White = srgb <1,1,1>;
 light_source {<40,350,300> White*1.5 spotlight point_at 0 radius 9 falloff 11}
 
 //--textures definitions--------------------------------
-
+#declare Black  = srgb <0,0,0>;
+#declare Gray60 = srgb <0.6,0.6,0.6>;
+#declare Gray80 = srgb <0.8,0.8,0.8>;
+#declare Gray90 = srgb <0.9,0.9,0.9>;
+#declare Red    = srgb <1,0,0>;
+#declare Tan    = srgb <0.86646,0.58571,0.44532>;
+#declare Wheat  = srgb <0.85494,0.85494,0.75836>;
```

Comment:
Maybe a standard like SVG or HTML colour names would provide an easy way out of the conundrum: If we provide a file "colors_web.inc" or some such, we can recommend that one for new scenes, use it in sample scenes, and deprecate the old "colors.inc".

Do you happen to know where SVG and HTML colours differ? (X11 colours are apparently not an option, since they are defined without reference to a particular colour space; SVG and HTML on the other hand are defined in terms of sRGB colours.)